api_volontaria/apps/volunteer/tests/test_view_events.py

```diff
@@ -94,7 +95,15 @@ def test_create_new_event_as_admin(self):
             "task_type": reverse(
                 'tasktype-detail',
                 args=[self.tasktype.id],
-            )
+            ),
+            "tags": [
+                {
+                    "name": "Tag 1",
+                },
+                {
+                    "name": "Tag 2",
+                },
+            ],
         }
 
         self.client.force_authenticate(user=self.admin)
```

Comment:
We should check if the tag is correctly added/created when we do that. 
Actually the test could pass even if the tag are not created by this method.